dashboard/views.py

```diff
@@ -593,11 +587,10 @@ def is_weight_considered(course_id):
     return value
 
 
-def get_term_dates_for_course(course_id):
-    logger.info(get_term_dates_for_course.__name__)
-    sql = "select a.date_start from course c, academic_terms a where c.id = %(course_id)s and c.term_id=a.id"
-    df = pd.read_sql(sql, conn, params={"course_id": course_id}, parse_dates={'date_start': '%Y-%m-%d'})
-    return df['date_start'].iloc[0]
+def get_course_date_start(course_id):
+    logger.info(get_course_date_start.__name__)
+    course_date_start = Course.objects.get(id=course_id).get_course_date_range()[0]
```

Comment:
Using namedtuple (above) would make this get_course_date_range().date_start

Reply:
See latest commit.